index.html

```diff
@@ -38,6 +38,138 @@ <h1>Hey there. I’m Promise <span class="title">I’m a software developer</spa
           </div>
         </div>
       </section>
+
+      <div class="go-to-works">
+        <a href="#works"><img src="./img/arrow.png" alt="arrow"></a>
+      </div>
+
+      <section id="works" class="grid">
+        <h2>My Recent Works</h2>
+        <hr>
+        <div class="card border">
+          <img src="./img/smartfacedetector.png" alt="smartfacedetector">
+          <div class="project-name">
+            <h4>My Smart Face Detector App</h4>
+          </div>
+          <div class="tech-group">
+            <span class="tech">Nodejs</span>
+            <span class="tech">css</span>
+            <span class="tech">Javascript</span>
+            <span class="tech">html</span>
+          </div>
```

Comment:
- [ ] It will be much better if you use `ul` tag to grab the list of technologies and use `li` tag for every technology. 
So instead of: 
```html
         <div class="tech-group">
            <span class="tech">Nodejs</span>
            <span class="tech">css</span>
            <span class="tech">Javascript</span>
            <span class="tech">html</span>
          </div>
```
Use 
```html
         <ul class="tech-group">
            <li class="tech">Nodejs</li>
            <li class="tech">css</li>
            <li class="tech">Javascript</li>
            <li class="tech">html</li>
          </ul>
```

Please apply this to all the lists you use. ` lines +67 to +72`, ` lines +80 to +85`, ` lines +106 to +111`, ` lines +119 to +124`, `lines +142 to +147`, `lines +152 to +158`, and ` lines +163 to +169`

Reply:
That's very true. Thank you.